packages/dart/sshnp_flutter/lib/src/presentation/widgets/custom_list_tile.dart

```diff
@@ -115,7 +115,7 @@ class CustomListTile extends StatelessWidget {
           }
           break;
         case CustomListTileType.faq:
-          final Uri url = Uri.parse('https://atsign.com/faqs/');
+          final Uri url = Uri.parse('https://docs.noports.com/ssh-no-ports/faq');
```

Comment:
If not there already, the faq document should include a comment noting that this url is referred to by the app.

Reply:
Can you elaborate more on this?

Reply:
Just that from now on, since the url https://docs.noports.com/ssh-no-ports/faq is now referred to from this app, a document must always be available at that url; and a good way to ensure that is true is to edit the faq doc noting that it is referred to by this app

Reply:
Thank Gary, @jenmonroe Please see Gary's comment above.

Reply:
I think we should create a redirect for this @gkc @cconstab ssh-desktop-faq.noports.com -> whatever url we use now or in the future